RECOVERY.md

```diff
@@ -159,30 +171,23 @@ bitcoin-cli -rpcwallet=recovery listtransactions
 Once those funds have been recovered and the refund delay has expired (the `confirmations` field of the previous command exceeds `25920`), you can send them to your normal on-chain wallet.
 Compute the total amount received (in our example, 1.5 BTC), choose the address to send to (for example, `bcrt1q9ez7rt33wynwpah582lnqlj3u0tpzsrkj2flas`) and create a transaction using all of the received funds:
 
-```sh
-bitcoin-cli -rpcwallet=recovery walletcreatefundedpsbt '[{"txid":"d9940b7eb709ff8eaec307bdd6d20633e30a6eb1627d9ef8c8e03dfd28298c75","vout":1,"sequence":25920}]' '[{"bcrt1q9ez7rt33wynwpah582lnqlj3u0tpzsrkj2flas":1.5}]' 0 '{"subtractFeeFromOutputs":[0]}'
-
+```shell
+bitcoin-cli -rpcwallet=recovery walletcreatefundedpsbt '[{"txid":"4c3236b1fa1f3ed124ab83b1667be95f855952e68729eae54a9f511c8c8cb993", "vout":0, "sequence":144}]' '[{"bcrt1qzy4h8dux6pjl8ys979632uynqffd53vjkzffjl":0.09}]'
```

Comment:
Yes, a blank wallet will not work (for now), it must also contain regular descriptors to generate changed addresses if needed.
Unfortunately we cannot yet use `sendtoaddress` (Bitcoin Core won't be able to sign) and have to create and sign PSBTs manually, but this will probably be simpler in future versions of Bitcoin Core.

Reply:
I indeed tested that `sendtoaddress` doesn't work (fails at signing time), but the PSBT procedure works when not using a blank wallet :+1: